### PR TITLE
Test cases translation scripts

### PR DESCRIPTION
**Pull request by ashutoshsingh0223**
Scripts for converting english test cases(yaml) to csv for translation and the back to yaml after translation.

### CODE REVIEW

File: test_case_translation/generate_translated_tests.py [hunk 1]
```diff
@@ -0,0 +1,153 @@
+# -*- coding: utf-8 -*-
+import io
```
Comment (srajdev) [inline test_case_translation/generate_translated_tests.py]: W1618 [34mimport missing `from __future__ import absolute_import`[0m

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
UNIT TESTS FAILED... Please resolve issues

**Comment by chiragjn** [comment]
retest this please

**Comment by haptik-deployment** [comment]
UNIT TESTS FAILED... Please resolve issues

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
UNIT TESTS FAILED... Please resolve issues

**Comment by ranvijayj** [comment]
retest this please

**Comment by haptik-deployment** [comment]
UNIT TESTS FAILED... Please resolve issues

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Pull request closed without merging**